### PR TITLE
Add test and Doc to set_parent function on Attached Component class

### PR DESCRIPTION
**Pull request by joaofoes**

### CODE REVIEW

File: docs/Components/AttachedComponent.md [hunk 1]
```diff
@@ -59,4 +59,16 @@ print(math.degrees(aileron.angle))
 plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
-```
\ No newline at end of file
+```
+
+## Set_parent
+
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:65]: Remove this blank line

File: docs/Components/AttachedComponent.md [hunk 2]
```diff
@@ -59,4 +59,16 @@ print(math.degrees(aileron.angle))
...
+The *set_parent* has already been used in the example above, but to give a further explanation, what the function it simply do is setting a parent to the current component.
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:66]: Typo:

"what the function does is to set a parent to the current component."

File: docs/Components/AttachedComponent.md [hunk 3]
```diff
@@ -59,4 +59,16 @@ print(math.degrees(aileron.angle))
...
+
+Note that if we try to set a parent to a component that has already a parent assigned it will return an error. In the example above, the aileron parent is the wing, see what happens if we try to set the plane as the aileron parent:
+
+``` python 
+
+aileron.set_parent(wing)
+>>> raise Exception('Component already has a parent: wing')
+```
```
Comment (rafaellehmkuhl) [inline docs/Components/AttachedComponent.md:74]: Nice! Errors should be explained too!

File: tests/Components/test_AttachedComponent.py [hunk 4]
```diff
@@ -25,3 +43,9 @@ def test_instantiation(attached_component):
 def test_states(attached_component):
     attached_component.actuation_angle = math.radians(3)
     assert(attached_component.actuation_angle == math.radians(3))
+
+def test_set_parent(attached_component, base_component, extra_base_component):
+    attached_component.set_parent(base_component)
+    assert(attached_component.parent == base_component)
+    with pytest.raises(Exception):
+        assert attached_component.set_parent(extra_base_component)
```
Comment (rafaellehmkuhl) [inline tests/Components/test_AttachedComponent.py:51]: Perfect

**Review by CassianoMontibeller — APPROVED** [review]

**Pull request merged**